packages/postgres-database/src/simcore_postgres_database/migration/versions/605fc07e068d_project_nodes_modification_creation_.py

```diff
@@ -0,0 +1,67 @@
+"""project_nodes modification + creation projects_node_to_pricing_unit
+
+Revision ID: 605fc07e068d
+Revises: b102946c8134
+Create Date: 2023-10-05 12:12:15.250040+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "605fc07e068d"
+down_revision = "b102946c8134"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    op.execute("ALTER TABLE projects_nodes DROP CONSTRAINT projects_nodes_pkey")
+    op.execute(
+        "ALTER TABLE projects_nodes ADD COLUMN project_node_id SERIAL PRIMARY KEY"
```

Comment:
so all rows will not automatically get a project_node_id?

Reply:
Yes by this command all rows will get project_node_id which will be prefilled automatically for the already created rows